### PR TITLE
Make channel mute command respond with and without ~ prefix

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
The channel /mute slash command now works with and without the `~` prefix

### CODE REVIEW

File: app/command_mute.go [hunk 1]
```diff
@@ -43,14 +43,20 @@ func (me *MuteProvider) DoCommand(a *App, args *model.CommandArgs, message strin
...
 	}
 
+	channelName := ""
+	splitMessage := strings.Split(message, " ")
 	// Overwrite channel with channel-handle if set
 	if strings.HasPrefix(message, "~") {
-		splitMessage := strings.Split(message, " ")
-		chanHandle := strings.Split(splitMessage[0], "~")[1]
-		data := (<-a.Srv.Store.Channel().GetByName(channel.TeamId, chanHandle, true)).Data
+		channelName = strings.Split(splitMessage[0], "~")[1]
```
Comment (crspeller) [inline app/command_mute.go]: Why split? You already checked the prefix so you can just do `splitMessage[0][1:]`

File: app/command_mute.go [hunk 2]
```diff
@@ -43,14 +43,20 @@ func (me *MuteProvider) DoCommand(a *App, args *model.CommandArgs, message strin
 		return &model.CommandResponse{Text: args.T("api.command_mute.error", map[string]interface{}{"Channel": channel.DisplayName}), ResponseType: model.COMMAND_RESPONSE_TYPE_EPHEMERAL}
 	}
 
+	channelName := ""
+	splitMessage := strings.Split(message, " ")
```
Comment (crspeller) [inline app/command_mute.go]: Maybe use splitN here because your only ever looking for the first part.

File: app/command_mute.go [hunk 3]
```diff
@@ -43,14 +43,20 @@ func (me *MuteProvider) DoCommand(a *App, args *model.CommandArgs, message strin
...
 	// Overwrite channel with channel-handle if set
 	if strings.HasPrefix(message, "~") {
```
Comment (cpanato) [inline app/command_mute.go]: can use this? or not `channelName := strings.TrimPrefix(message, "~")`

File: app/command_mute.go (hunk 1 above)
Comment (enahum) [reply app/command_mute.go]: done

File: app/command_mute.go (hunk 3 above)
Comment (enahum) [reply app/command_mute.go]: this won't work cause I want to avoid things like `~channelName and more stuff`

Comment (cpanato) [reply app/command_mute.go]: ok, thanks!

**Pull request merged**